### ISSUE TITLE
re-add greenkeeper into master

### ISSUE THREAD

**Issue by codekiln** [codekiln/beetimer#63]
#1 looked like it was solved, but actually, it wasn't. Deleting the `greenkeer/initial` branch and then removing and re-adding the association recreated it, but we're still getting build failures having to do with jsx/a11y: https://github.com/codekiln/beetimer/pull/62: 

```
$ yarn build
yarn build v0.17.8
$ node scripts/build.js 
Creating an optimized production build...
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.
Failed to compile.
./src/Firebase.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/index.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/service-worker-registration.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/MainLayout.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/TopBar.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/App.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/UserAvatar.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/Tracks/TracksList.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/Tracks/TrackMessage.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/Tracks/TrackCardView.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
./src/Tracks/TrackCardEdit.js
  Line 1:  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
error Command failed with exit code 1.
```
